test/_regtest.ts

```diff
@@ -39,3 +39,17 @@ export async function fetchTxHex(txId: string): Promise<string> {
   }
   return hex;
 }
+
+export async function mint(address: string, quantity: number): Promise<string> {
```

Comment:
We should return also the tx hash along with the asset

Reply:
Theorically, after mint we can fetch the txid with fetchUtxos